ckan/config/middleware/flask_app.py

```diff
@@ -173,6 +173,7 @@ def hello_world_post():
 
     # Auto-register all blueprints defined in the `views` folder
     _register_core_blueprints(app)
+    _register_plugin_blueprints(app)
```

Comment:
@smotornyuk i tried as this, not sure if this is as your idea. Please check it and give some thoughts

Reply:
Yep, you've got the idea, but there are small problems bellow